api/pkg/common/common.go

```diff
@@ -142,6 +143,19 @@ func GetFilter(request *restful.Request, filterOpts []string) (map[string]string
 	return filter, nil
 }
 
+func GetFilterPathParams(request *restful.Request, filterOpts []string) (map[string]string, error) {
```

Comment:
Can we use GetFilter, instead create this new function?

Reply:
GetFilter is for QueryParams. Please check this link.
https://github.com/sodafoundation/multi-cloud/blob/development/api/pkg/common/common.go#L132

For this API, we want to get the PathParam filter for the BackendId